src/components/LayerMetadataPopupService.js

```diff
@@ -50,16 +57,19 @@ goog.require('ga_wms_service');
             className: 'ga-tooltip-metadata ga-popup-tablet-full',
             x: 400,
             y: 200,
-            showPrint: true
+            showPrint: true,
+            lang: $translate.use()
```

Comment:
This where you initialize the language in the popup so yes I would say it s needed (scope.popup.options.lang in tooltip directive)